ops/verifier.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  verifier:
+    depends_on:
+      - l1_chain
+      - deployer
+    image: ethereumoptimism/verifier
+    build:
+      context: ..
+      dockerfile: ./ops/docker/Dockerfile.geth
+    # override with the geth script and the env vars required for it
+    entrypoint: sh ./geth.sh
+    env_file:
+      - ./envs/geth.env
+    environment:
+        ETH1_HTTP: http://l1_chain:8545
+        ROLLUP_STATE_DUMP_PATH: http://deployer:8081/state-dump.latest.json
+        # used for getting the addresses
+        URL: http://deployer:8081/addresses.json
+        # connecting to the DTL
+        # TODO(annieke): should this be different for verifier?
+        ROLLUP_CLIENT_HTTP: http://dtl:7878
```

Comment:
@tynes i haven't gotten this running yet, would connecting the verifier to the same dtl be a problem?

Reply:
I think it should work

Reply:
When https://github.com/ethereum-optimism/optimism/pull/552 lands then I know it will definitely work. Pre #552 then it should work as long as the DTL default backend is L1